scripts/lib/setup_venv.py

```diff
@@ -108,14 +106,28 @@ def get_venv_dependencies(vendor: str, os_version: str) -> List[str]:
     else:
         raise AssertionError("Invalid vendor")
 
+
 def install_venv_deps(pip: str, requirements_file: str, python2: bool) -> None:
-    pip_requirements = os.path.join(ZULIP_PATH, "requirements", "pip2.txt" if python2 else "pip.txt")
-    run([pip, "install", "--force-reinstall", "--require-hashes", "--requirement", pip_requirements])
+    pip_requirements = os.path.join(
+        ZULIP_PATH, "requirements", "pip2.txt" if python2 else "pip.txt",
+    )
+    run(
+        [
+            pip,
+            "install",
+            "--force-reinstall",
+            "--require-hashes",
+            "--requirement",
+            pip_requirements,
+        ],
+    )
```

Comment:
Do you understand why this got exploded?

Reply:
Oh, I see, it's 102 characters.  I'm not really a fan of the `[`s on their own lines strategy here.

Reply:
This is an explosion that I’m merely okay with, as an instance of a simple general rule that does well enough in most cases. To the extent that an independent justification is desired, you could imagine it being helpful for diffs like this:

```diff
     run(
         [
             pip,
             "install",
             "--force-reinstall",
             "--require-hashes",
             "--requirement",
             pip_requirements,
         ],
+        cwd="/srv/zulip-py3-venv",
+        env={
+            **os.environ,
+            "PATH": "/srv/zulip-py3-venv/bin:/usr/local/bin:/usr/bin:/bin",
+            "BLAH": "blah,blah,blah",
+        },
      )
```